### PR TITLE
Update interactivetool_wallace.xml

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: tools/interactive/interactivetool_wallace.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
         </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
     </environment_variables>
     <command><![CDATA[
-        ln -s '${infile}' /srv/shiny-server/data/inputdata.txt &&
+        ln -s '${infile}' /import/ &&
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml]: Does the IT load all files in this folder? How does it know the name of the file?

File: tools/interactive/interactivetool_wallace.xml [hunk 2]
```diff
@@ -24,15 +24,15 @@
...
         /usr/bin/shiny-server.sh
     ]]>
     </command>
     <inputs>
-        <param name="infile" type="data" format="tabular,txt" label="Tabular in Clarion format"/>
+        <param name="infile" type="data" format="tabular,txt,csv,rdata" label="text oriented (tabular or csv) or RData files"/>
     </inputs>
     <outputs>
-        <!--data name="outfile" format="txt" /-->
+        <data name="outfile" format="tabular,txt,csv,rdata" />
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml]: This IT does not produce yet and output ... isn't it?

File: tools/interactive/interactivetool_wallace.xml (hunk 1 above)
Comment (yvanlebras) [reply tools/interactive/interactivetool_wallace.xml]: Very good question ;) in fact, there is several GUI modules on this it and at each step the user can load data. In the GIE configuration we were using Galaxy helpers to scan history datasets and proposing the user to select a file from his history and import it on /import/. So here I think that if a file is on /import/ we can find it though a R dedicated gui function but here I need to go more into details to test it. So for now the idea is to indicate the /import/ folder and I will check this more deeply I hope today

File: tools/interactive/interactivetool_wallace.xml (hunk 2 above)
Comment (yvanlebras) [reply tools/interactive/interactivetool_wallace.xml]: And yes, this IT produces output ;) as at each RShiny module you can export datasets resulting in computation. But here again I need to test things and commits I hope today

File: tools/interactive/interactivetool_wallace.xml (hunk 1 above)
Comment (bgruening) [reply tools/interactive/interactivetool_wallace.xml]: Keep in mind the old galaxy_helper script do still work and is imho the preferred way to get more data or write back data to your history. You do not need to have an input or an output at all, if you don't want to ...

See the Askomics IT ... they also load via the API

Comment (yvanlebras) [reply tools/interactive/interactivetool_wallace.xml]: Yes, Exactly what I have in mind! But I need to pay attention to each step and be sure all is ok. So  this means I will ""reactivate"" the use of galaxy_helpers on this Docker image in, I hope ;), some minutes, but I need to check things are ok for you. The more complicated thing is here I don't test this docker image in a Galaxy context, so activating galaxy_helper give me error like "can't connect to Galaxy instance" to have Galaxy API infos... I will have a look at askomics but if you have special advice on the use of galaxy_helper script in IT scope, don't hesitate ;)

File: tools/interactive/interactivetool_wallace.xml [hunk 3]
```diff
@@ -24,15 +24,14 @@
         </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
     </environment_variables>
     <command><![CDATA[
-        ln -s '${infile}' /srv/shiny-server/data/inputdata.txt &&
         /usr/bin/shiny-server.sh
     ]]>
     </command>
     <inputs>
-        <param name="infile" type="data" format="tabular,txt" label="Tabular in Clarion format"/>
+        <param name="infile" type="data" format="csv" label="csv file with for first columns: 'name', 'longitude', 'latitude' informations"/>
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml]: but then also remove this input and the output

Comment (yvanlebras) [reply tools/interactive/interactivetool_wallace.xml]: yes, I'm on it! ;)

**Comment by bgruening** [comment]
Ok, let me take this in ... and rebuild the entire thing.

**Pull request merged**